### PR TITLE
feat: support end-to-end KT LoRA serving for Qwen3.5 MoE

### PR DESCRIPTION
**Pull request by jdai0**
## Summary

This PR adds the SGLang runtime side of the Qwen3.5 MoE KT LoRA serving workflow.

Main changes:

- Add KT expert LoRA serving support for Qwen3.5 MoE expert modules.
- Add Qwen3.5 non-expert LoRA support for attention / shared modules.
- Allow a merged KT LoRA adapter passed through `--lora-paths name=<adapter_dir>` to be detected and split internally at server startup.
- Route expert LoRA weights to the KT CPU expert path and non-expert LoRA weights to SGLang's LoRA manager.
- Fix Triton LoRA B masking for non-aligned tile shapes.

## Motivation

KT-SFT for Qwen3.5 MoE produces LoRA weights that need two runtime paths:

- expert LoRA through the KT CPU expert forward path
- non-expert LoRA through SGLang's LoRA manager

Earlier serving required users to manually pass separate expert and non-expert adapter paths. With this change, users can pass one merged adapter through `--lora-paths`, while SGLang handles the expert/non-expert split internally.

## Test Plan

- Served a converted Qwen3.5 MoE KT-SFT merged adapter with `--lora-paths <name>=<merged_adapter_dir>`.
- Verified startup logs show internal expert/non-expert adapter preparation.
- Verified OpenAI-compatible chat completion with `model=<served_model>:<adapter_name>`.
- Verified base-model requests still use `weight_version=default` and do not require LoRA.

### CODE REVIEW

File: python/sglang/srt/layers/moe/kt_ep_wrapper.py [hunk 1]
```diff
@@ -104,6 +107,164 @@ class KTConfig:
...
+    hidden_size: int,
+    moe_intermediate_size: int,
+    dtype: torch.dtype = torch.bfloat16,
+) -> KTExpertLoraWeights:
+    adapter_dir = Path(adapter_path)
+    rank_from_config, alpha = _load_adapter_config(adapter_dir)
+    weight_file = _find_adapter_weight_file(adapter_dir)
+    from safetensors.torch import load_file
+
+    state_dict = load_file(str(weight_file), device="cpu")
```
[the remark this answers is not in the document]
Comment (jdai0) [reply python/sglang/srt/layers/moe/kt_ep_wrapper.py]: Resolved in 79da01c

File: python/sglang/srt/server_args.py [hunk 2]
```diff
@@ -73,6 +75,169 @@
...
+    digest.update(str(adapter_dir).encode())
+    for path in (weight_path, config_path):
+        stat = path.stat()
+        digest.update(str(stat.st_mtime_ns).encode())
+        digest.update(str(stat.st_size).encode())
+    cache_root = (
+        Path(os.environ.get("SGLANG_KT_LORA_CACHE_DIR", ""))
+        if os.environ.get("SGLANG_KT_LORA_CACHE_DIR")
+        else Path(tempfile.gettempdir()) / "sglang_kt_lora_cache"
+    )
```
[the remark this answers is not in the document]
Comment (jdai0) [reply python/sglang/srt/server_args.py:175]: Resolved in 79da01c

File: python/sglang/srt/models/qwen3_5.py [hunk 3]
```diff
@@ -576,6 +879,40 @@ def self_attention(
...
+                        dtype=qkv.dtype,
+                        device=qkv.device,
+                    )
+                if v_delta is None:
+                    v_delta = torch.zeros(
+                        qkv.shape[:-1] + (self.kv_size,),
+                        dtype=qkv.dtype,
+                        device=qkv.device,
+                    )
+                qkv = qkv + torch.cat((q_delta, k_delta, v_delta), dim=-1)
```
[the remark this answers is not in the document]
Comment (jdai0) [reply python/sglang/srt/models/qwen3_5.py]: Resolved in 79da01c
